app/helpers/pages_helper.rb

```diff
@@ -14,6 +14,10 @@ def format_ak_ui_url(api_url, ak_ui_url)
     api_url.gsub(%r{.*rest\/v1}, ak_ui_url)
   end
 
+  def ak_resource_id(ak_resource_url)
+    /\/([^\/]*)\/$/.match(ak_resource_url)[1]
```

Comment:
I feel like this would be stronger if it specifically matched digits `[0-9]` rather than just not slashes `[^\/]` and it didn't require a trailing slash (as the URL is still valid without a trailing slash, and AK could change that without warning)